src/playlist.rs

```diff
@@ -140,28 +141,28 @@ pub struct VariantStream {
     pub codecs: Option<String>,
     pub resolution: Option<String>,
     pub frame_rate: Option<String>,
-    pub hdcp_level: Option<String>,
+    pub hdcp_level: Option<QuotedOrUnquoted>,
```

Comment:
Is `frame_rate` (for example) defined to always be a quoted string?

Reply:
`frame_rate` (and everything else) I believe is not changed.

Reply:
Yeah I mean according to the spec, what kind of string is it supposed to be?

Reply:
well... :) ok, I will check that, but I wasn't going to review the whole spec in the PR! just the piece I found breaking CLOSED-CAPTIONS, lol

Reply:
That's OK, then let's create a separate issue for going over all the attributes afterwards. Various of those can also be parsed into more exact types according to the spec.

E.g. for the framerate

```
      The value is a decimal-floating-point describing the maximum frame
      rate for all the video in the Variant Stream, rounded to three
      decimal places.
```

This should really become an `Option<f32>` or so instead of a string.

Reply:
```
FRAME-RATE

      The value is a decimal-floating-point describing the maximum frame
      rate for all the video in the Variant Stream, rounded to three
      decimal places.
```

So, yes, it shouldn't be rendered quoted. It's parsed as `QuotedOrUnquoted::Unquoted` and then stored as String
```
    pub frame_rate: Option<String>,
/..
            frame_rate: attrs.remove("FRAME-RATE").map(|f| f.to_string()),
```
 and then rendered without quotes:

```
    fn write_stream_inf_common_attributes<T: Write>(&self, w: &mut T) -> std::io::Result<()> {
//...
        write_some_attribute!(w, ",FRAME-RATE", &self.frame_rate)?;
        write_some_attribute!(w, ",HDCP-LEVEL", &self.hdcp_level)?;
    }
```

...which is the same for all the fields that have `String` or `Option<String>` type

The only change is for `CLOSED-CAPTIONS` and `HDCP-LEVEL` because their type is changed

```
    pub hdcp_level: Option<QuotedOrUnquoted>,
//...
    pub closed_captions: Option<QuotedOrUnquoted>,
```

Reply:
Yeah, and those two should have more specific types for the "enumerated strings" and I think then we're OK with this PR, and then someone will have to clean up all the attribute types further afterwards before the next release.

Reply:
I totally missed this comment... Sorry... Hm... That makes a bit more sense, actually.

Reply:
I agree that it's a good idea, but let's do this in another PR.